config/routes.rb

```diff
@@ -7,6 +7,6 @@
     get "static_pages/help"
     get "/signup", to: "users#new"
     post "/signup", to: "users#create"
-    resources :users, only: %i(show new create)
+    resources :users, only: %i(index show new create edit update destroy)
```

Comment:
dùng hết rồi thì bỏ only nha
```suggestion
    resources :users
```